packages/metal-quartz-button/src/MetalQuartzButton.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import MetalQuartzButtonBase from './MetalQuartzButtonBase';
+import core from 'metal';
+import Soy from 'metal-soy';
+
+import templates from './MetalQuartzButton.soy.js';
+
+/**
+ * Metal Quartz Button component.
+ */
+class MetalQuartzButton extends MetalQuartzButtonBase {
+}
+
+/**
+ * State definition.
+ * @static
+ * @type {!Object}
+ */
+MetalQuartzButton.STATE = {
```

Comment:
What about icons? We can reuse the already existing [`quartz-icon` component](https://github.com/metal/metal-quartz-components/tree/master/packages/metal-quartz-icon)